src/+replab/+equi/PhasedMatrixPartition.m

```diff
@@ -0,0 +1,434 @@
+classdef PhasedMatrixPartition < replab.Obj
+% Describes a generalized partition of a matrix in disjoint blocks
```

Comment:
I think I got confused here because I was expecting blocks to refer to matrix blocks similar to what appears when discussing [block matrices](https://en.wikipedia.org/wiki/Block_matrix). But we really mean an arbitrary subset of matrix elements (i.e. without the constraint of being a square matrix by itself), right?

Would the terminology of mask or basis be clearer?

Reply:
Indeed, good point (though it's a pain to rename everything, I'll do it).
I'll go for "subset" instead of "block"

Reply:
Done!